scripts/ci/testing/summarize_junit_failures.py

```diff
@@ -32,45 +32,62 @@
 
 
 @lru_cache(maxsize=None)
-def translate_classname(classname):
-    # The pytest xunit output has "classname" in the python sense as 'dir.subdir.package.Class' -- we want to
-    # convert that back in to a pytest "selector" of dir/subdir/package.py::Class
+def translate_classname(classname: str) -> str | None:
+    """
+    Translates a classname in the form 'dir.subdir.package.Class' to a pytest selector in the form
+    'dir/subdir/package.py::Class'.
+
+    Args:
+        classname: The classname to translate.
+
+    Returns:
+        The translated pytest selector.
```

Comment:
We generally use Sphinx-style derivatives instead of Google-style. See other functions for examples.